lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -0,0 +1,267 @@
+"""
+Tests for Courses
+"""
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from courseware.tests.factories import UserFactory
+from xmodule.modulestore.tests.factories import CourseFactory
+from student.models import CourseEnrollment
+from student.views import login_oauth_token
+from social.apps.django_app.default.models import UserSocialAuth
+import httpretty
+import json
+from openedx.core.djangoapps.user_api.api.profile import preference_info, update_preferences
+
+
+class TestGroups(ModuleStoreTestCase, APITestCase):
+    """
+    Tests for /api/mobile/v0.5/courses/...
+    """
+
+    USERNAME_1 = "Daniel Eidan"
+    EMAIL_1 = "daniel@ebnotions.com"
+    PASSWORD_1 = "edx"
+    FB_ID_1 = "10155110991745300"
+
+    USERNAME_2 = "Marc Ashman"
+    EMAIL_2 = "marc@ebnotions.com"
+    PASSWORD_2 = "edx"
+    FB_ID_2 = "10154833899435243"
+
+    USERNAME_3 = "Peter Organa"
+    EMAIL_3 = "peter@ebnotions.com"
+    PASSWORD_3 = "edx"
+    FB_ID_3 = "10154805420820176"
+
+    BACKEND = "facebook"
+    USER_URL = "https://graph.facebook.com/me"
+    UID_FIELD = "id"
+
+    # TODO: Needs to be a valid access token
+    _FB_USER_ACCESS_TOKEN = 'CAAKbz9eIdVsBABmFt9kSO34MkLI0AwuLemGbwXLgxoYbmTXuh1sKIuGoZAjeK1XdIHMBoURsll0iq1OG7Jpz0B1iHuk4OYvhSgJdFihaNqOkHM8HHlNXjTUODjUn3ol5s4lYDP5NpDR1wUHCocZCBUWuZABBW3BNR5oDwypTVjAU7OjXnTZCBagsGuv1CEKPTIE5EcvUanQmuJEE02Ta'
```

Comment:
Why does this have to be a valid access token?
Doesn't the `httppretty.activate` decorator mock these FB API calls?